lua/starfall/libs_sh/pvp.lua

```diff
@@ -56,64 +64,89 @@ function pvp_library.getBuilders()
 end
 
 -- isInPvp
+--- Returns true if the player is in Pvp mode, false otherwise.
+-- @param ply Player
+-- @return True or False
```

Comment:
maybe something like `Boolean, is the player is in pvp`?

same for the build and other pvp function

Reply:
👍 Done